### ISSUE TITLE
Fix batchNormalization Layer 

### ISSUE THREAD

**Issue by MokkeMeguru** [MokkeMeguru/TFGENZOO#1]
ref. https://github.com/tensorflow/probability/blob/r0.8/tensorflow_probability/python/bijectors/batch_normalization.py#L72-L259

but I don't know is this layer correct.

**Comment by MokkeMeguru** [comment]
WIP
